package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "poynt",
-  "version": "0.0.26",
+  "version": "0.0.27",
```

Comment:
oops. i pushed a release. should be `0.0.28` now

Reply:
fixed.